### DISCUSSION TITLE
Code Mode

### DISCUSSION THREAD

**Discussion by TFinotti** [monkeytypegame/monkeytype#1006] [Ideas] [answered]
I believe that typing code has a very unique pattern, using different kinds of brackets and punctuation much more frequently than any other kind of text. This feature would allow developers to practice typing in different programming languages.

First implementation idea that comes to mind are small (5 - 10) lines of code snippets. Maybe common data structures and algorithms implementations?

Please share your thoughts!

**Comment by Miodec** [comment, marked as answer]
You can get code snippets by changing the language to the one you'd like to type and selecting quote mode (all though there isnt that many code quotes in the lists at the moment)
